test/main/test.js

```diff
@@ -11,16 +11,65 @@ if ( typeof document !== "undefined" ) {
 
 	QUnit.module( "fixture", function( hooks ) {
 		var failure = false,
-			values = [ /* initial value (see unshift below), */ "<b>ar</b>", undefined ],
+			values = [
+
+				/* initial value (see unshift below), */
+				/* initial value (see unshift below), */
```

Comment:
Confused as to why this comment is duplicated.

Reply:
The comments represent items in the `values` array here, and we now push the original value twice (the two `values.unshift` calls below).

Reply:
Oh, okay, now I understand. (Kind of a weird setup for these tests, though...)

Reply:
INDEED! It was massively confusing for a bit, I _tried_ to leave them a little easier to grok for next time but I definitely agree it is tricky as heck...